androidApp/src/main/AndroidManifest.xml

```diff
@@ -3,16 +3,15 @@
     <application
                  android:allowBackup="true"
                  android:supportsRtl="true"
-                 android:theme="@style/AppTheme">
+                 android:theme="@style/BlueberryTheme">
```

Comment:
Note: color theme serialization has not been implemented yet.

Reply:
This is using Android's built-in theming mechanism.